client/tasks/fills/tax/index.js

```diff
@@ -77,11 +76,11 @@ class Tax extends Component {
 	}
 
 	isTaxJarSupported() {
-		const { countryCode, tasksStatus } = this.props;
+		const { countryCode, task } = this.props;
 		const {
 			automatedTaxSupportedCountries = [],
```

Comment:
This should be renamed to `automatedSupportCountries`

Reply:
Good catch!  Updated this to `woocommerceTaxCountries` given the changes occurring in https://github.com/woocommerce/woocommerce-admin/pull/7874